src/app/components/SharePopover/index.js

```diff
@@ -84,25 +81,27 @@ const ShareLink = ({ link }) => {
   `;
 };
 
-
 export default SharePopover;
 
 const LABELS = {
-  email: '',
+  email: 'Email',
   facebook: 'Facebook',
-  native: '',
+  native: 'More options',
   copylink: 'Copy',
-  linkedin: '',
-  twitter: 'X (formerly Twitter)',
+  linkedin: 'LinkedIn',
+  twitter: 'X (formerly Twitter)'
 };
 
-
 export async function trackShare(socialNetwork) {
   const { url } = await getMeta();
   dataLayer.event('share', { socialNetwork, url });
 }
 
-function native({ id, url, title, description }) {
-  navigator.share({ text: description, title, url }).then(() => trackShare(id));
+function nativeShare({ id, url, title, description }) {
+  navigator
+    .share({ text: description, title, url })
```

Comment:
Apparently [this isn't available in the Android webview](https://devdocs.io/dom/navigator/share), is that an issue? An async function with a try/catch block would handle this case.